de.tubs.variantsync.core/src/de/tubs/variantsync/core/patch/base/DefaultDelta.java

```diff
@@ -101,8 +101,8 @@ public void setRevisedFromString(String revised) {
 
 	@Override
 	public String getRepresentation() {
-		return "--- (" + this.original.getPosition() + ") " + this.original.getLines() + "\n" + "+++ (" + this.revised.getPosition() + ") "
-			+ this.revised.getLines();
+		return "--- (" + this.original.getPosition() + ") " + this.original.getLines() + "\n" + "+++ ("
+				+ this.revised.getPosition() + ") " + this.revised.getLines();
```

Comment:
This is not OS-safe, change `\n` to `String.format("%n");`

Reply:
Or maybe `System.lineSeparator()`?

Reply:
This problem will be addressed by the following issue https://github.com/tthuem/VariantSync/issues/56